docs/install_guides/pypi-openvino-dev.md

```diff
@@ -113,34 +113,34 @@ For example, to install and configure the components for working with TensorFlow
 
 ## What's in the Package?
 
-> **NOTE**: The openvino-dev package installs [OpenVINO™ Runtime](https://pypi.org/project/openvino) as a dependency, which is the engine that runs the deep learning model and includes a set of libraries for an easy inference integration into your applications.  
+> **NOTE**: The openvino-dev package installs [OpenVINO™ Runtime](https://pypi.org/project/openvino) as a dependency, which is the engine that runs the deep learning model and includes a set of libraries for an easy inference integration into your applications.
 
 **In addition, the openvino-dev package installs the following components by default:**
 
-| Component        | Console Script                                                                   | Description                                                                                                                                                                                                                                                                                                   |  
+| Component        | Console Script                                                                   | Description                                                                                                                                                                                                                                                                                                   |
 |------------------|---------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| [Model conversion API](https://docs.openvino.ai/nightly/openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html) | `mo` |**Model conversion API** imports, converts, and optimizes models that were trained in popular frameworks to a format usable by OpenVINO components. <br>Supported frameworks include Caffe\*, TensorFlow\*, MXNet\*, PaddlePaddle\*, and ONNX\*.                                               |
+| [Model conversion API](https://docs.openvino.ai/nightly/openvino_docs_OV_Converter_UG_Deep_Learning_Model_Optimizer_DevGuide.html) | `mo` |**Model conversion API** imports, converts, and optimizes models that were trained in popular frameworks to a format usable by OpenVINO components. <br>Supported frameworks include Caffe\*, TensorFlow\*, MXNet\*, PaddlePaddle\*, and ONNX\*.                                               |
```

Comment:
Wrong link to ovc/convert_model. In the context, it should be a link to legacy `mo` docs.